### PR TITLE
[DOCS-5848] Add Java instructions for CI Visibility ITR

### PR DESCRIPTION
**Pull request by nikita-tkachenko-datadog**
### What does this PR do?

Add instructions on how to configure CI Visibility Intelligent Test Running with Java.
Also does some minor improvements in CI Visibility Tests for Java page.

### Motivation

Intelligent Test Runner for Java is about to enter Public Beta mode.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-5848 to review

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Intelligent Test Runner for Java
+kind: documentation
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:3]:
```suggestion
kind: documentation
is_beta: true
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+further_reading:
+    - link: "/continuous_integration/tests"
+      tag: "Documentation"
+      text: "Explore Test Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+<div class="alert alert-warning">Intelligent Test Runner for Java is a beta product.</div>
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:13]:
```suggestion
{{< callout url="#" >}}Intelligent Test Runner for Java is a beta product.{{< /callout >}} 
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Compatibility
+
+Intelligent Test Runner is supported in `dd-java-agent >= 1.18.0`
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:17]:
```suggestion
Intelligent Test Runner is supported in `dd-java-agent >= 1.18.0`.
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+## Setup
+
+Prior to setting up Intelligent Test Runner, set up [Test Visibility for Java][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
+
+To enable Intelligent Test Runner, set the following environment variables:
+
+`DD_APPLICATION_KEY` (Required)
+: The [Datadog Application key][2] used to query the tests to be skipped.<br/>
+If you are reporting data through the Agent, then it is the agent process that will require the key.<br/>
+If you are using agentless mode, then they key will be used by the tracer process.<br/>
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:28]:
```suggestion
If you are reporting data through the Agent, then the agent process requires the key.<br/>
If you are using Agentless mode, then the tracer process uses the key.<br/>
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+**Default**: `(empty)`<br/>
+
+`DD_CIVISIBILITY_JACOCO_PLUGIN_VERSION` (Optional)
+: Intelligent test runner requires [Jacoco][4] for collecting code coverage.<br/>
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:32]:
```suggestion
: Intelligent Test Runner requires [Jacoco][4] for collecting code coverage.<br/>
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+Set this variable to a valid Jacoco version (e.g. `0.8.10`) if you want the tracer run your build with Jacoco injected.<br/>
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:33]:
```suggestion
Set this variable to a valid Jacoco version (such as `0.8.10`) if you want the tracer to run your build with Jacoco injected.<br/>
```

File: content/en/continuous_integration/intelligent_test_runner/java.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+Omit it if your project already has Jacoco configured.<br/>
+**Default**: `(empty)`
+
+`DD_CIVISIBILITY_JACOCO_PLUGIN_INCLUDES` (Optional)
+: Use this variable to configure the list of packages for which code coverage should be collected.<br/>
+It is optional, but setting this helps to reduce overhead by avoiding coverage collection for core JDK classes and 3rd party dependencies.<br/>
```
Comment (alai97) [inline content/en/continuous_integration/intelligent_test_runner/java.md:39]:
```suggestion
While this variable is optional, setting this helps reduce overhead by avoiding coverage collection for core JDK classes and third-party dependencies.<br/>
```

File: content/en/continuous_integration/tests/java.md [hunk 8]
```diff
@@ -48,6 +48,17 @@ To report test results to Datadog, you need to configure the Datadog Java librar
...
+Additionally, configure [Datadog site][2] to which you want to send data.
+
+`DD_SITE` (Required)
+: The [Datadog site][2] to upload results to.<br/>
+**Default**: `datadoghq.com`<br/>
+**Selected site**: {{< region-param key="dd_site" code="true" >}}
+
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: /getting_started/site/
```
Comment (alai97) [inline content/en/continuous_integration/tests/java.md:60]: I added this reusable shortcode that we can use instead: https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/ci-agentless.md!

```suggestion
```

File: content/en/continuous_integration/tests/java.md [hunk 9]
```diff
@@ -57,57 +68,26 @@ You only need to download the tracer library once for each server.
...
-curl https://repo1.maven.org/maven2/com/datadoghq/dd-java-agent/$DD_TRACER_VERSION/dd-java-agent-$DD_TRACER_VERSION.jar --output $DD_TRACER_FOLDER/dd-java-agent-$DD_TRACER_VERSION.jar
+wget -O $DD_TRACER_FOLDER/dd-java-agent.jar https://dtdg.co/latest-java-tracer
 {{< /code-block >}}
 
-[1]: https://mvnrepository.com/artifact/com.datadoghq/dd-java-agent
-[2]: https://img.shields.io/maven-central/v/com.datadoghq/dd-java-agent?style=flat-square
-
-{{% /tab %}}
-{{< /tabs >}}
+You can run `java -jar $DD_TRACER_FOLDER/dd-java-agent.jar` command to check the version of the tracer library
```
Comment (alai97) [inline content/en/continuous_integration/tests/java.md:83]:
```suggestion
You can run the `java -jar $DD_TRACER_FOLDER/dd-java-agent.jar` command to check the version of the tracer library.
```

**Review by alai97 — COMMENTED** [review]
Some copy suggestions + beta banner nit!

**Review by alai97 — APPROVED** [review]

**Pull request merged**